terminus-ui/card/src/card.component.spec.ts

```diff
@@ -1,95 +1,139 @@
+  let cardComponent: TsCardComponent;
+  let cardComponent2: TsCardComponent;
 
   beforeEach(() => {
-    component = new TsCardComponent();
-    component2 = new TsCardComponent();
+    fixture = createComponent(TestHostComponent, [], [TsCardModule]);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+    cardComponent = component.cardComponent;
+    card = fixture.debugElement.query(By.css('.c-card')).nativeElement as HTMLElement;
+    cardComponent2 = new TsCardComponent();
   });
 
+  describe(`isDisabled`, () => {
+    test(`card is not disabled`, () => {
+      expect(cardComponent.disabled).toEqual(false);
+      component.disabled = false;
+      fixture.detectChanges();
+      expect(cardComponent.isDisabled).toEqual(false);
+      expect(cardComponent.disabled).toEqual(false);
+      expect(card.classList).not.toContain('c-card--disabled');
+    });
 
-  it(`should exist`, () => {
-    expect(component).toBeTruthy();
+    test(`card is disabled`, () => {
+      component.disabled = true;
+      fixture.detectChanges();
+      expect(cardComponent.isDisabled).toEqual(true);
+      expect(cardComponent.disabled).toEqual(true);
+      expect(card.classList).toContain('c-card--disabled');
+    });
   });
 
-  describe(`centeredContent`, () => {
-    it(`should set and retrieve`, () => {
-      component.centeredContent = true;
-      expect(component.centeredContent).toEqual(true);
+  describe(`isFlat`, () => {
+    test(`card is flat`, () => {
+      component.flat = true;
+      fixture.detectChanges();
+      expect(cardComponent.flat).toEqual(true);
+      expect(card.classList).toContain('c-card--flat');
+    });
+    test(`card is not flat`, () => {
+      component.flat = false;
+      fixture.detectChanges();
+      expect(cardComponent.flat).toEqual(false);
+      expect(card.classList).not.toContain('c-card--flat');
     });
   });
 
   describe(`supportsInteraction`, () => {
-    test(`should set and retrieve`, () => {
+    test(`card supports interaction`, () => {
       component.supportsInteraction = true;
-      expect(component.supportsInteraction).toEqual(true);
+      fixture.detectChanges();
+      expect(cardComponent.supportsInteraction).toEqual(true);
+      expect(card.classList).toContain('c-card--interaction');
+    });
+    test(`card does not support interaction`, () => {
+      component.supportsInteraction = false;
+      fixture.detectChanges();
+      expect(cardComponent.supportsInteraction).toEqual(false);
+      expect(card.classList).not.toContain('c-card--interaction');
     });
   });
 
+  describe(`centeredContent`, () => {
+    it(`should set and retrieve`, () => {
+      expect(cardComponent.centeredContent).toEqual(false);
+      cardComponent.centeredContent = true;
+      expect(cardComponent.centeredContent).toEqual(true);
+    });
+  });
 
   describe(`set aspectRatio()`, () => {
-
     it(`should convert the string aspect to a percentage`, () => {
```

Comment:
I'd like to see this test actually verifying the DOM changes..Not terrible dissimilar from the last comment:

```html
<ts-card [aspectRatio]="myAspect" id="outer">
  Content
</ts-card>
```

Test:

```typescript
// Set first ratio
component.myAspect = '1:1';
// Get the dimensions
let dimensions = document.getElementById('outer').getBoundingClientRect();
// Verify 1 to 1 ratio
expect(dimensions.width).toEqual(dimensions.height);

// Set new ratio
component.myAspect = '16:9';
// Get the new dimensions
dimensions = document.getElementById('outer').getBoundingClientRect();
// Verify widescreen aspect
expect(dimensions.width).toBeGreaterThan(dimensions.height);
```